### PR TITLE
upgrade to arc v28

### PR DESCRIPTION
**Pull request by dkent600**
GenesisScheme => DaoCreator
remove "Log" from events
_globalconstraint => _globalConstraint in AddGlobalConstraint

### CODE REVIEW

File: lib/dao.js [hunk 1]
```diff
@@ -288,12 +288,12 @@ export class DAO {
   /**
    * Returns promise of the DAOstack (Genesis) avatar address, or undefined if not found
    */
-  static async getDAOstack(genesisSchemeAddress) {
+  static async getDaoCreator(daoCreatorAddress) {
```
Comment (orenyodfat) [inline lib/dao.js]: @dkent600 What this function is doing ? Are you sure it is has the right name ?

File: migrations/2_deploy_organization.js [hunk 2]
```diff
@@ -44,14 +44,14 @@ let schemeGCRegisterParams;
...
 
 module.exports = async function (deployer) {
-  // Deploy GenesisScheme:
+  // Deploy DaoCreator:
   // apparently we must wrap the first deploy call in a then to avoid
   // what seem to be race conditions during deployment
-  // await deployer.deploy(GenesisScheme)
-  deployer.deploy(GenesisScheme, { gas: 6015000 }).then(async () => {
-    genesisSchemeInst = await GenesisScheme.deployed();
+  // await deployer.deploy(DaoCreator)
```
Comment (orenyodfat) [inline migrations/2_deploy_organization.js]: Please remove commented code

**Comment by orenyodfat** [comment]
Is there a test dedicated to DaoCreator ?

**Comment by dkent600** [comment]
@orenyodfat 

> Is there a test dedicated to DaoCreator ?

I didn't actually create any because it is getting used every time we call `DAO.new` in the tests so I figured at this moment that was good enough.  Maybe I was being lazy....let me know.

File: lib/dao.js (hunk 1 above)
Comment (orenyodfat) [reply lib/dao.js]: @dkent600 What this function is doing ?

**Comment by orenyodfat** [comment]
> I didn't actually create any because it is getting used every time we call DAO.new in the tests so I figured at this moment that was good enough. Maybe I was being lazy....let me know.

I guess it is good to have a dedicated standalone (as possible) test for each lib module.
I think that we should strive that the automated tests should cover each and every line of code in the lib.

**Pull request merged**